lib/data/monorepo.json

```diff
@@ -350,7 +351,7 @@
       "https://github.com/mui/material-ui",
       "https://github.com/mui/mui-x"
     ],
-    "mdc-react": "material-components/material-components-web-react",
+    "mdc-react": "https://github.com/material-components/material-components-web-react",
```

Comment:
this fix should be in a separate PR

Reply:
https://github.com/renovatebot/renovate/pull/30917